site/src/docs/components/header/code.mdx

```diff
@@ -291,15 +287,11 @@ import { Header, IconSearch } from 'hds-react';
           />,
         ]}
       />
-      <Header.Link
-        as={Link}
-        href="#"
-        label="Kasvatus ja koulutus"
-      />
+      <Header.Link as={Link} href="#" label="Kasvatus ja koulutus" />
       <Header.Link as={Link} href="#" label="Asuminen" />
     </Header.NavigationMenu>
   </Header>
-)
+);
```

Comment:
All of the code example's are broken.
`SyntaxError: Unexpected token (64:1)
60 :       <Header.Link as={Link} href="#" label="Kasvatus ja koulutus" />
61 :       <Header.Link as={Link} href="#" label="Asuminen" />
62 :     </Header.NavigationMenu>
63 :   </Header>
64 : );)`

Reply:
Saving and auto-formatting mdx-files adds sometimes ";" to the end on lines on my machine which sometimes breaks these 😢